packages/connector-aws-ses/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "logto-connertor-aws",
+  "version": "1.0.0",
+  "description": "Logto Connector for AWS",
```

Comment:
`Amazon SES` or `SES` would be more accurate? We may have other connectors for AWS in the future